src/main/views/choose-action.njk

```diff
@@ -10,12 +10,7 @@
 {% endblock %}
 
 {% block beforeContent %}
-  {{ super() }}
-
-  {{ govukBackLink({
-    text: "Back",
-    href: "/"
-  }) }}
+<a href="/" class="govuk-back-link" onclick="goBack(event);">Back</a>
```

Comment:
We need to put this in the json language files. The Welsh trasnlation is 'Yn ôl'